content/verein/transparency.fr.md

```diff
@@ -0,0 +1,84 @@
+{
+    "title": "Transparence",
+    "slug": "transparence",
+    "type": "page"
+}
+
+<a href="https://www.transparency.de/mitmachen/initiative-transparente-zivilgesellschaft/" class="no-link-decoration"><img src="/img/logo-itz.svg" style="float: right; width: 40%; min-width: 200px; padding: 5px;"></a>
+
+Afin que vous puissiez nous faire confiance, nous nous sommes engagés à être aussi transparents que possible sur nos procédures, nos finances et nos décisions. C'est pourquoi nous avons adhéré à l'[Initiative for a Transparent Civil Society](https://www.transparency.de/mitmachen/initiative-transparente-zivilgesellschaft/), une initiative de la branche allemande de Transparency International, qui est autorisée à nous contrôler par échantillonnage. Ils garantissent le respect de notre promesse de gestion transparente. Sur cette page, nous avons résumé toutes les informations importantes que nous publions dans le cadre de cette initiative.
```

Comment:
As far as I remember, "Initiative for a Transparent Civil Society" is just a translation by us. I think the only official name is the German "Initiative Transparente Zivilgesellschaft". So, do feel free to translate that name into French for easier readability.